docs/dev/table/connect.md

```diff
@@ -500,6 +501,39 @@ rowtime:
 The following watermark strategies are supported:
 
 <div class="codetabs" markdown="1">
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+-- Sets a watermark strategy for strictly ascending rowtime attributes. Emits a watermark of the
+-- maximum observed timestamp so far. Rows that have a timestamp smaller to the max timestamp
+-- are not late.
+CREATE TABLE MyTable (
+  ts_field TIMESTAMP(3),
+  WATERMARK FOR ts_field AS ts_field
+) WITH (
+  ...
+)
+
+-- Sets a watermark strategy for ascending rowtime attributes. Emits a watermark of the maximum
+-- observed timestamp so far minus 1. Rows that have a timestamp equal to the max timestamp
+-- are not late.
+CREATE TABLE MyTable (
+  ts_field TIMESTAMP(3),
+  WATERMARK FOR ts_field AS ts_field - INTERVAL '0.001' SECOND
+) WITH (
+  ...
+)
+
+-- Sets a watermark strategy for rowtime attributes which are out-of-order by a bounded time interval.
```

Comment:
It is same to above? Why are out-of-order?

Reply:
We don't have a shortcut API for this. 
All the watermark strategies are expressed by expressions, we didn't provide built-in strategy functions. So the `asceding timestamp` is a special case of `out-of-order timestamp` which has a constant 1 ms delay.

Reply:
It is same to above example... Why different description?

Reply:
The difference is that `ascending timestamp` must use `INTERVAL '0.001' SECOND`.

Reply:
Can we change to 2s? We can keep same to Java/Scala/Python.